app/src/main/res/drawable/ic_filter_sort.xml

```diff
@@ -0,0 +1,5 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android" android:height="24dp" android:tint="?colorControlNormal" android:viewportHeight="24" android:viewportWidth="24" android:width="24dp">
```

Comment:
Please also format that file using Android studio's builtin formatter. It's just for convenience.